cmd/xl-v1-object.go

```diff
@@ -249,7 +249,11 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 	chunkSize := getChunkSize(xlMeta.Erasure.BlockSize, xlMeta.Erasure.DataBlocks)
 	pool := bpool.NewBytePool(chunkSize, len(onlineDisks))
 
-	// Read from all parts.
+	storage, err := NewErasureStorage(onlineDisks, xlMeta.Erasure.DataBlocks, xlMeta.Erasure.ParityBlocks, xl.readQuorum, xl.writeQuorum)
```

Comment:
Why don't we take out read & write quorum when creating the object, and instead pass it is explicity for ReadFile (just read quorom of course) and into CreateFile (just writeQuorum).
Think this is cleaner and also a bit more flexible.